build/release/build_release.sh

```diff
@@ -139,11 +139,17 @@ ant -f rename.xml -Drelease=$tag -Dseries=$series
 popd > /dev/null
 
 # build the release
-
 if [ "$SKIP_BUILD" != true ]; then
-  echo "building release"
   export MAVEN_OPTS="-Xmx2048m"
-  mvn $MAVEN_FLAGS -DskipTests -Dall clean -Pcollect install
+  echo "building release: clean"
+  mvn $MAVEN_FLAGS -Dall clean
```

Comment:
May need to remove $MAVEN_FLAGS reference here, not sure what is being passed in (it it includes a -T setting for example)